operational/flexera/custom_branding/README.md

```diff
@@ -0,0 +1,75 @@
+# Custom Branding
+
+## What It Does
+
+This Policy allows a customer to brand the FlexeraOne platform quickly and consistently.
+
+## How It Works
+
+- Run the Custom Branding policy in the tenant ensuring you select the credentials required
+- For non MSP Customers you can only apply the All Customer Settings Category

```

Comment:
Use indented list items (spaces before the -) where appropriate to make things more readable. Lines 10 and 11 look like they could be a sublist.

Reply:
Unresolved because issue was not addressed.